src/plugins/orangehrmCorporateDirectoryPlugin/Api/EmployeeDirectoryAPI.php

```diff
@@ -56,6 +56,37 @@ class EmployeeDirectoryAPI extends Endpoint implements CrudEndpoint
     ];
 
     /**
+     * @OA\Get(
+     *     path="/api/v2/directory/employees/{empNumber}",
+     *     tags={"CorporateDirectory/Employee Directory"},
```

Comment:
```suggestion
     *     tags={"Directory"},
```
:thinking: same as menu item